src/test/java/com/rackspace/salus/monitor_management/services/TestMonitorServiceTest.java

```diff
@@ -78,7 +82,7 @@
   private static final Duration resultsTimeout = Duration.ofMillis(500);
 
   @Configuration
-  @Import({TestMonitorService.class})
+  @Import({TestMonitorService.class, MetricsAutoConfiguration.class, CompositeMeterRegistryAutoConfiguration.class})
```

Comment:
This one had the inner config class, which threw off your pattern :)
```suggestion
  @Import({TestMonitorService.class, SimpleMeterRegistry.class})
```